doc/source/serve/direct-ingress.md

```diff
@@ -0,0 +1,102 @@
+# Direct Ingress
+
+In the 2.1, Serve provides an alpha version of gRPC ingress. In this section, you will learn how to
+
+* use Serve's internal gRPC schema to send traffic
+* bring your own gRPC schema into your Serve application
+
```

Comment:
add a note saying this documentation assumes some basic understanding of gRPC

Reply:
Or have a link to the gRPC documentation.

Reply:
added a grpc documentation above directly